wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/DocumentInfoScreen.kt

```diff
@@ -246,24 +255,29 @@ fun DocumentInfoScreen(
                     text = { Text(text = stringResource(R.string.document_info_screen_menu_item_delete)) },
                     leadingIcon = { Icon(Icons.Outlined.Delete, contentDescription = null) },
                     onClick = {
-                        showMenu = false
+                        showMenu = false;
                         showDeleteConfirmationDialog = true
                     }
                 )
+                DropdownMenuItem(
+                    text = { Text(text = stringResource(R.string.document_info_screen_menu_activities_log)) },
+                    leadingIcon = { Icon(Icons.Outlined.Info, contentDescription = null) },
+                    onClick = {
+                        onNavigate(WalletDestination.DocumentInfo.getRouteWithArguments(
+                            listOf(
+                                Pair(WalletDestination.DocumentInfo.Argument.DOCUMENT_ID, documentInfo.documentId),
+                                Pair(WalletDestination.DocumentInfo.Argument.SECTION, "activities")
+                            )
+                        ))
+                        showMenu = false
+                    }
+                )
```

Comment:
I think this menu item should be after "Show Credentials" and before "Delete", so move it up one.

Reply:
Done.